web/src/main/webapp/script.js

```diff
@@ -12,6 +12,48 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function searchMe() {
+  var input, filter, ul, li, item, i, txtValue;
```

Comment:
I actually think these should be of type `const` not `var`. This is because these actual variables aren't changing. Also it is unclear to me why you define `item` out here and not in the for loop.